README.md

```diff
@@ -2,12 +2,47 @@
 
 ---
 
-> TechStack - React JS, Material UI
+## Installation
 
-_MORE INFO YET TO BE ADDED... STAY TUNED_
+The Open Event Frontend can be easily deployed on a variety of platforms, so just run the following commands when setting up for the first time:
 
-# DON'T JUST CLONE, PLEASE LEAVE A STAR, MOTIVATES US TO ADD MORE FEATURES TO THE PROJECT.
+- `yarn install or npm install  `
+- `yarn start or npm start `
+- Visit your app at [http://localhost:3000](http://localhost:3000).
```

Comment:
After this line , ask the user to fetch the NEW-UI branch from the origin and the create a branch from it, as staging and master are protected branches, All PRs to Staging/Master will not be considered.